reference/enterprise-api-ref/export-import-api.markdown

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Import & Export API
+published: true
+tags: [reference, enterprise, API, import, export]
+---
+
+Import & Export API allows to transfer Mission portal data between hubs.
+
+## Get available items to export
+
+This API call provides a list of items available for export. Please note, roles affect access, some items like settings require the admin role for export access.
+
```

Comment:
Please note that the role of the user that authenticates to this API will affect what items are available. For example: the API user must have admin role in order to export settings.

Reply:
done